charts/cmc-citizen-frontend/Chart.yaml

```diff
@@ -1,7 +1,7 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
try to rebase with master now ... otherwise you will get conflict

Reply:
rebase with master